std/string.d

```diff
@@ -359,11 +359,17 @@ alias CaseSensitive = Flag!"caseSensitive";
     Params:
         s = string or InputRange of characters to search in correct UTF format
         c = character to search for
+        startIdx = starting index to a well-formed code point
         cs = CaseSensitive.yes or CaseSensitive.no
 
     Returns:
-        the index of the first occurrence of $(D c) in $(D s). If $(D c)
+        the index of the first occurrence of $(D c) in $(D s) with
+        respect to the start index $(D startIdx). If $(D c)
         is not found, then $(D -1) is returned.
+        If $(D c) is found the value of the returned index is at least
+        $(D startIdx).
+        If the sequence starting at $(D startIdx) does not represent a well
+        formed codepoint, then a $(XREF utf,UTFException) may be thrown.
```

Comment:
*May* be thrown? Not *is* thrown? It probably also belongs in the `Thrown:` section